utilities/manual/source/background.md

```diff
@@ -82,6 +82,41 @@ Note: The most common tests are run automatically, if there were changes
 to the executable (i.e. TXS has been compiled since the last run).
 Furthermore all tests are run once a week.
 
+## Menu Definitions
+
+Menu definitions are used to set-up menus. You can [configure the Menu]{configuration.md#configuring-the-menu-(advanced-option)} to
+have menus defined by you. Of course the existing standard menu is defined in this way. The definition is located in the source
+file `texstudio/uiconfig.xml`. This shows an excerpt:
+
+```xml
+<TexMakerXUI>
+<menu id="main/latex" text="&amp;LaTeX">
+  <insert id="documentclass" text="\documentclass" insert="\documentclass[%&lt;options%|%>]{%&lt;class%>}"/>
+  <insert id="usepackage" text="\usepackage{}" insert="\usepackage[%&lt;options%>]{%&lt;package%|%>}"/>
+  ...
+  
+  <menu id="sectioning" text="&amp;Sectioning">
+    <insert id="part" text="part" insert="\part{%&lt;title%>}" icon="part"/>
+    <insert id="chapter" text="chapter" insert="\chapter{%&lt;title%>}" icon="chapter"/>
+    ...
+  </menu>
+
+  ...
+
+</menu>
+</TexMakerXUI>
+```
+
+The `\<TexMakerXUI\>` tag is mandatory and contains one ore more `\<menu\>` tags. Each menu must have a unique `id` attribute. The
```

Comment:
There is no mention that this is xml, so reference to a tutorial for that ?

Probably a better way to explain part, is to quote them and explain them directly afterwards, possibly with an image what the end result looks like.